client/js/app.js

```diff
@@ -5,18 +5,27 @@ import openingHours from './components/opening-hours';
 
 message();
 
-const burgerEl = document.querySelector('.js-header-burger');
-const headerSearchEl = document.getElementById('header-search');
-const openingHoursEls = document.querySelectorAll('.js-opening-hours');
+const init = () => {
+  const burgerEl = document.querySelector('.js-header-burger');
+  const headerSearchEl = document.getElementById('header-search');
+  const openingHoursEls = document.querySelectorAll('.js-opening-hours');
 
-if (burgerEl) {
-  headerBurger(burgerEl);
-}
+  if (burgerEl) {
+    headerBurger(burgerEl);
+  }
 
-if (headerSearchEl) {
-  headerSearch(headerSearchEl);
-}
+  if (headerSearchEl) {
+    headerSearch(headerSearchEl);
+  }
 
-if (openingHoursEls) {
-  openingHours(openingHoursEls);
+  if (openingHoursEls) {
+    openingHours(openingHoursEls);
+  }
+};
+
+if (document.readyState !== 'loading') {
+  init();
+} else {
+  document.addEventListener('DOMContentLoaded', init());
```

Comment:
Should this be the function call `init()`? I think it be just the reference to the function `init` (or called within an anonymous function)?